src/test/ui/issues/issue-75704.rs

```diff
@@ -0,0 +1,7 @@
+// Caused an infinite loop during SimlifyCfg MIR transform previously.
+//
+// build-pass
```

Comment:
does this really test this mir pass? don't you also need `-Zmir-opt-level=2`?

Reply:
For this test case no additional flags are necessary to reproduce the issue. Unlike the one from #75051 which also needed `-Zmir-opt-level=2`.

Reply:
ah, superb! thanks